### PR TITLE
Update redirect

### PR DESCRIPTION
**Pull request by sukantoraymond**
# Docs PR Template

## Why this should be merged

<please give a thorough description of why these changes are necessary>

## How this works

<please give a thorough description of what your changes are and how they work>

## How these changes were tested 

<please give a thorough description of how to view the changes and how they were tested>

## To prevent any errors while building

- [x] run `vale /docs/file/path` on all changed `.md` files to ensure all grammar rules pass
- [x] run `markdownlint /docs/file/path` on all changed `.md` files to ensure all linting rules pass
- [x] complete the above two checks and all additional rules outlined in `style-checker-notes.md` to
  ensure all checks pass

### If a document was removed/deleted

- [x] the path to that doc must be redirected to a valid URL via the
  `_redirects` file
- [x] `_redirects` were manually verified with the cloudflare preview link
- [x] `sidebars.json` reflects all changes made to file path

### If a document was moved

- [x] all files that were moved from their current directory to a new path have had their paths
  redirected via the `_redirects` file
- [x] `_redirects` were manually verified with the cloudflare preview link
- [x] `sidebars.json` reflects all changes made to file path

### CODE REVIEW

**Review by meaghanfitzgerald — CHANGES_REQUESTED** [review]
Thanks for adding the changes to the redirects file. Unfortunately this doesn't fix the CLI docs links in the other `.md` files. 

If you run `yarn build`, the terminal shows you all the broken links and their locations:

```
[WARNING] Docusaurus found broken anchors!

Please check the pages of your site in the list below, and make sure you don't reference any anchor that does not exist.
Note: it's possible to ignore broken anchors with the 'onBrokenAnchors' Docusaurus configuration, and let the build pass.

Exhaustive list of all broken anchors found:
- Broken anchor on source page path = /build/cross-chain/awm/overview:
   -> linking to /learn/avalanche/subnets-overview#virtual-machines
- Broken anchor on source page path = /build/cross-chain/teleporter/overview:
   -> linking to #teleporter-protocol (resolved as: /build/cross-chain/teleporter/overview#teleporter-protocol)
- Broken anchor on source page path = /build/subnet:
   -> linking to /learn/avalanche/subnets-overview#virtual-machines
- Broken anchor on source page path = /build/subnet/deploy/fuji-testnet-subnet:
   -> linking to /learn/avalanche/subnets-overview#virtual-machines
   -> linking to /nodes/run/node-manually#connect-to-fuji-testnet
   -> linking to /nodes/validate/what-is-staking#fuji-testnet
   -> linking to /build/subnet/deploy/local-subnet#create-a-custom-subnet-configuration
   -> linking to /nodes/validate/add-a-validator#add-a-validator-with-avalanche-wallet
- Broken anchor on source page path = /build/subnet/deploy/mainnet-subnet:
   -> linking to /nodes/validate/add-a-validator#add-a-validator-with-avalanche-wallet
- Broken anchor on source page path = /build/subnet/info/wagmi:
   -> linking to /build/subnet/info/wagmi#adding-wagmi-to-metamask
- Broken anchor on source page path = /build/subnet/upgrade/considerations-subnet-upgrade:
   -> linking to /nodes/configure/avalanchego-config-flags#snow-sample-size-int
- Broken anchor on source page path = /build/subnet/upgrade/customize-a-subnet:
   -> linking to /reference/subnet-evm/api#ethgetchainconfig
- Broken anchor on source page path = /build/vm/create/rust-vm:
   -> linking to /nodes/configure/avalanchego-config-flags#vm-configs
- Broken anchor on source page path = /build/vm/golang-vms/golang-vm-simple:
   -> linking to /nodes/configure/avalanchego-config-flags#vm-configs
- Broken anchor on source page path = /build/vm/rust-vms/installing-vm:
   -> linking to /nodes/configure/avalanchego-config-flags#vm-configs
- Broken anchor on source page path = /deprecated/tutorials-contest/2022/local-subnet-development:
   -> linking to /build/subnet/utility/deploy-smart-contract-to-subnet#step-1-setting-up-metamask
- Broken anchor on source page path = /learn/avalanche/subnets-overview:
   -> linking to /nodes/run/node-manually#requirements
- Broken anchor on source page path = /nodes/maintain/setting-up-node-monitoring:
   -> linking to /nodes/run/third-party/aws-node#f8df
- Broken anchor on source page path = /nodes/run/third-party/google-cloud-node:
   -> linking to #setup-instructions (resolved as: /nodes/run/third-party/google-cloud-node#setup-instructions)
- Broken anchor on source page path = /nodes/validate/what-is-staking:
   -> linking to /intro#avax
- Broken anchor on source page path = /reference/avalanchego/admin-api:
   -> linking to /build/vm/intro#load-a-vm
- Broken anchor on source page path = /reference/avalanchego/p-chain/txn-format:
   -> linking to /reference/standards/cryptographic-primitives#secp-256-k1-addresses
- Broken anchor on source page path = /reference/avalanchego/x-chain/api:
   -> linking to /learn/avalanche/avalanche-platform#exchange-chain-x-chain
- Broken anchor on source page path = /reference/avalanchego/x-chain/txn-format:
   -> linking to /reference/standards/cryptographic-primitives#secp-256-k1-addresses
- Broken anchor on source page path = /reference/standards/guides/issuing-api-calls:
   -> linking to /nodes/configure/chain-config-flags#--http-host-string
- Broken anchor on source page path = /reference/standards/guides/x-chain-migration:
   -> linking to /reference/avalanchego/index-api#x-chain-vertices
- Broken anchor on source page path = /tooling/avalanchejs-overview:
   -> linking to /#Requirements
- Broken anchor on source page path = /tooling/cli-create-deploy-subnets/deploy-fuji-testnet-subnet:
   -> linking to /learn/avalanche/subnets-overview#virtual-machines
   -> linking to /nodes/run/node-manually#connect-to-fuji-testnet
   -> linking to /nodes/validate/what-is-staking#fuji-testnet
   -> linking to /build/subnet/deploy/local-subnet#create-a-custom-subnet-configuration
   -> linking to /nodes/validate/add-a-validator#add-a-validator-with-avalanche-wallet
- Broken anchor on source page path = /tooling/cli-create-deploy-subnets/deploy-mainnet-subnet:
   -> linking to /nodes/validate/add-a-validator#add-a-validator-with-avalanche-wallet
```

You will have to go into these files and correct these links.

File: static/_redirects [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+
+/tooling/cli-guides/create-a-validator-aws.md /tooling/cli-create-nodes/create-a-validator-aws.md
+/tooling/cli-guides/create-a-validator-gcp.md /tooling/cli-create-nodes/create-a-validator-gcp.md
+/tooling/cli-guides/node-ssh.md /tooling/cli-create-nodes/node-ssh.md docs
+/tooling/cli-guides/setup-a-devnet.md /tooling/cli-create-nodes/setup-a-devnet.md
+/tooling/cli-guides/stop-node.md /tooling/cli-create-nodes/stop-node.md
+/tooling/cli-guides/upload-a-custom-vm-to-cloud.md /tooling/cli-create-nodes/stop-node.md
+/tooling/cli-guides/validate-primary-network.md /tooling/cli-create-nodes/stop-node.md
+/tooling/cli-guides/validate-subnets.md /tooling/cli-create-nodes/stop-node.md
+
```
Comment (meaghanfitzgerald) [inline static/_redirects:11]: All of these redirects should be at the end of this file.

File: static/_redirects [hunk 2]
```diff
@@ -1,3 +1,14 @@
+# 02/06/2024
+
+/tooling/cli-guides/create-a-validator-aws.md /tooling/cli-create-nodes/create-a-validator-aws.md
+/tooling/cli-guides/create-a-validator-gcp.md /tooling/cli-create-nodes/create-a-validator-gcp.md
+/tooling/cli-guides/node-ssh.md /tooling/cli-create-nodes/node-ssh.md docs
```
Comment (meaghanfitzgerald) [inline static/_redirects:5]:
```suggestion
/tooling/cli-guides/node-ssh.md /tooling/cli-create-nodes/node-ssh.md
```

File: static/_redirects [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
+/tooling/cli-guides/setup-a-devnet.md /tooling/cli-create-nodes/setup-a-devnet.md
+/tooling/cli-guides/stop-node.md /tooling/cli-create-nodes/stop-node.md
+/tooling/cli-guides/upload-a-custom-vm-to-cloud.md /tooling/cli-create-nodes/stop-node.md
+/tooling/cli-guides/validate-primary-network.md /tooling/cli-create-nodes/stop-node.md
+/tooling/cli-guides/validate-subnets.md /tooling/cli-create-nodes/stop-node.md
```
Comment (meaghanfitzgerald) [inline static/_redirects:10]:
```suggestion
/tooling/cli-guides/create-a-validator-aws.md /tooling/cli-create-nodes/create-a-validator-aws.md 301
/tooling/cli-guides/create-a-validator-gcp.md /tooling/cli-create-nodes/create-a-validator-gcp.md 301
/tooling/cli-guides/node-ssh.md /tooling/cli-create-nodes/node-ssh.md 301
/tooling/cli-guides/setup-a-devnet.md /tooling/cli-create-nodes/setup-a-devnet.md 301
/tooling/cli-guides/stop-node.md /tooling/cli-create-nodes/stop-node.md 301
/tooling/cli-guides/upload-a-custom-vm-to-cloud.md /tooling/cli-create-nodes/stop-node.md 301
/tooling/cli-guides/validate-primary-network.md /tooling/cli-create-nodes/stop-node.md 301
/tooling/cli-guides/validate-subnets.md /tooling/cli-create-nodes/stop-node.md 301
```

File: static/_redirects [hunk 4]
```diff
@@ -556,3 +556,14 @@
...
+# 02/06/2024
+
+/tooling/cli-guides/create-a-validator-aws.md /tooling/cli-create-nodes/create-a-validator-aws.md 301
+/tooling/cli-guides/create-a-validator-gcp.md /tooling/cli-create-nodes/create-a-validator-gcp.md 301
+/tooling/cli-guides/node-ssh.md /tooling/cli-create-nodes/node-ssh.md 301
+/tooling/cli-guides/setup-a-devnet.md /tooling/cli-create-nodes/setup-a-devnet.md 301
+/tooling/cli-guides/stop-node.md /tooling/cli-create-nodes/stop-node.md 301
+/tooling/cli-guides/upload-a-custom-vm-to-cloud.md /tooling/cli-create-nodes/stop-node.md 301
+/tooling/cli-guides/validate-primary-network.md /tooling/cli-create-nodes/stop-node.md 301
+/tooling/cli-guides/validate-subnets.md /tooling/cli-create-nodes/stop-node.md 301
```
Comment (meaghanfitzgerald) [inline static/_redirects:569]:
```suggestion
# 02/06/2024

/tooling/cli-guides/create-a-validator-aws /tooling/cli-create-nodes/create-a-validator-aws 301
/tooling/cli-guides/create-a-validator-gcp /tooling/cli-create-nodes/create-a-validator-gcp 301
/tooling/cli-guides/node-ssh /tooling/cli-create-nodes/node-ssh 301
/tooling/cli-guides/setup-a-devnet /tooling/cli-create-nodes/setup-a-devnet 301
/tooling/cli-guides/stop-node /tooling/cli-create-nodes/stop-node 301
/tooling/cli-guides/upload-a-custom-vm-to-cloud /tooling/cli-create-nodes/stop-node 301
/tooling/cli-guides/validate-primary-network /tooling/cli-create-nodes/stop-node 301
/tooling/cli-guides/validate-subnets /tooling/cli-create-nodes/stop-node 301
```

**Comment by meaghanfitzgerald** [comment]
@sukantoraymond , you created these 2 docs last week
/tooling/cli-create-deploy-subnets/deploy-fuji-testnet-subnet.md
/tooling/cli-create-deploy-subnets/deploy-mainnet-subnet.md

there should not be any broken links. Idk where you copied these from, but for instance:
```
Broken anchor on source page path = /tooling/cli-create-deploy-subnets/deploy-fuji-testnet-subnet:
   -> linking to /learn/avalanche/subnets-overview#virtual-machines
```
our subnets-overview.md [page](https://docs.avax.network/learn/avalanche/subnets-overview) has not had a "Virtual Machines" subsection for months. This link is _in fact_ broken because when u click it, it dumps you on the "Subnets" page which has 0 information about virtual machines

:arrow_right: fix the broken links on your pages, and make sure every link is formatted with a .md . 
Less is more. Don't include links in your docs unless they are necessary.

**Comment by laviniat1996** [comment]
closing this in favor to https://github.com/ava-labs/avalanche-docs/pull/1666

**Pull request closed without merging**